.circleci/config.yml

```diff
@@ -248,6 +248,27 @@ jobs:
               exit 1
             fi
 
+  check-pull-number:
+    docker:
+      - image: cimg/base:2022.10
+    steps:
+      - checkout
+      - run:
+          name: Check that 'Pull#XXXX' matches PR number
+          command: |
+            COMMITS="$(git log --format=format:%B master.."$CIRCLE_SHA1")"
+            
```

Comment:
Move it validation.sh or similar sh file

Reply:
done